.github/workflows/ci.yml

```diff
@@ -45,7 +45,13 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v4
+      - uses: antontroshin/setup-go@bda02de8887c9946189f81e7e59512914aeb9ea4
```

Comment:
To play safe, this is a fork commit, and somehow I feel we could raise the fix to the upstream maybe, instead of using the fork from someone else.

I understand the benefits, but I'm a bit concerned about opening this

Reply:
OTOH, we are using `permissions: contents: read` by default. But according to our guideline, we need to be cautious to use third-party actions.

Reply:
We can give a go; the only thing could be the one whether the commit vanished